modules/twinklewarn.js

```diff
@@ -297,63 +367,63 @@ Twinkle.warn.messages = {
 				label: "Page moves against naming conventions or consensus",
 				summary: "General note: Page moves against naming conventions or consensus"
 			},
-			"uw-tpv1": {
-				label: "Refactoring others' talk page comments",
-				summary: "General note: Refactoring others' talk page comments"
-			},
-			"uw-upload1": {
-				label: "Uploading unencyclopedic images",
-				summary: "General note: Uploading unencyclopedic images"
-			}
-		}/*,
-		"To be removed from Twinkle": {
 			"uw-redirect1": {
 				label: "Creating malicious redirects",
 				summary: "General note: Creating malicious redirects"
 			},
-			"uw-ics1": {
-				label: "Uploading files missing copyright status",
-				summary: "General note: Uploading files missing copyright status"
+			"uw-speedy1": {
+				label: "Removing speedy deletion tags",
+				summary: "General note: Removing speedy deletion tags"
 			},
-			"uw-af1": {
-				label: "Inappropriate feedback through the Article Feedback Tool",
-				summary: "General note: Inappropriate feedback through the Article Feedback Tool"
+			"uw-tpv1": {
+				label: "Refactoring others' talk page comments",
+				summary: "General note: Refactoring others' talk page comments"
 			}
-		}*/
+		}
 	},
 
 
 	level2: {
 		"Common warnings": {
 			"uw-vandalism2": {
-				label: "Vandalism",
-				summary: "Caution: Unconstructive editing"
+				label: "Blatant vandalism",
+				summary: "Caution: Blatant vandalism"
```

Comment:
I think that this is still too strong for level 2, and belongs to level 3.

I'd fix this after merging instead of waiting for a fix here.